pytestqt/qtbot.py

```diff
@@ -234,7 +234,7 @@ def stopForInteraction(self):
 
     stop = stopForInteraction
 
-    def waitSignal(self, signal=None, timeout=1000, raising=None):
+    def waitSignal(self, signal=None, timeout=1000, callback=None, raising=None):
```

Comment:
I think this parameter should appear at the end, so code doing `waitSignal(signal, 1000, True)` won't break.